<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createForm</class>
 <widget class="QDialog" name="createForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formulaire</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="title">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Cr√©ation de formulaire - EMH</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="message">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::FieldsStayAtSizeHint</enum>
         </property>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Information</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPlainTextEdit" name="messageInformation">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>75</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Important</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPlainTextEdit" name="messageImportant">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>75</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPlainTextEdit" name="description">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>75</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Description*</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="nomLineEdit">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="nomLabel">
           <property name="text">
            <string>Nom*</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>Demande</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QRadioButton" name="radioButton_2">
           <property name="text">
            <string>Information</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Couleur</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="writerLayout">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Utilisateurs*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollArea" name="writersArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWriters">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>180</width>
              <height>271</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="readerLayout">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Lecteurs*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollArea" name="readersArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaReaders">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>180</width>
              <height>271</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="tagsLayout">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Tags</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollArea" name="tagsArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaTags">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>180</width>
              <height>271</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="fieldsLayout"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="addFieldButton">
         <property name="text">
          <string>Ajouter un champs</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="validButton">
         <property name="text">
          <string>Valider</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>createForm</receiver>
   <slot>chooseColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addFieldButton</sender>
   <signal>clicked()</signal>
   <receiver>createForm</receiver>
   <slot>addField()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>705</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>632</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>validButton</sender>
   <signal>clicked()</signal>
   <receiver>createForm</receiver>
   <slot>valid()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>771</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>704</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>chooseColor()</slot>
  <slot>valid()</slot>
  <slot>addField()</slot>
 </slots>
</ui>
